zerver/models.py

```diff
@@ -1053,6 +1054,27 @@ def parse_usermessage_flags(val):
         mask <<= 1
     return flags
 
+class Attachment(models.Model):
+    MAX_FILENAME_LENGTH = 100
+    file_name = models.CharField(max_length=MAX_FILENAME_LENGTH, db_index=True)
+    path_id = models.TextField(default='/some/path')
```

Comment:
Oh I forgot to remove the default value. I added that just for debugging purposes.